reo/models.py

```diff
@@ -194,6 +203,7 @@ class LoadProfileModel(models.Model):
     outage_is_major_event = models.BooleanField(default=True)
 
     #Outputs
+    year_one_electric_load_series_kw_bau = ArrayField(models.FloatField(null=True, blank=True), default=list)
```

Comment:
I don't think that we need this output? it is the same as the `loads_kw` with no outage, and the same as `year_one_electric_load_series_kw` with an outage (see https://github.com/NREL/REopt_Lite_API/blob/4e45d564260f605b63c507b24753b3a0d7769ebc/reo/src/load_profile.py#L496).

Reply:
This has been removed in commit 036435b22aada8a3e485ec07cdd319979db22321